### PR TITLE
Developer App 1.6.2 Release Post

### PR DESCRIPTION
**Pull request by timkim**

### CODE REVIEW

File: _posts/blog/2016-04-18-1.6.2-Release.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+author: "Tim Kim"
+title: "PhoneGap Developer App 1.6.x Update"
```
Comment (macdonst) [inline _posts/blog/2016-04-18-1.6.2-Release.md]: Should be 1.6.2

File: _posts/blog/2016-04-18-1.6.2-Release.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+tags:
+- Release
+- App
+- Windows Phone
+- Android
+- iOS
+---
+
+Hello everyone, we have updated the PhoneGap Developer App to version 1.6.x. In this version, you will find that we have updated:
```
Comment (macdonst) [inline _posts/blog/2016-04-18-1.6.2-Release.md]: Should be 1.6.2

Comment (timkim) [inline _posts/blog/2016-04-18-1.6.2-Release.md]: Ya, I wasn't sure. It's 1.6.0 for android and windows and 1.6.2 for ios.

Comment (macdonst) [inline _posts/blog/2016-04-18-1.6.2-Release.md]: Oh weird. It would be nice to have the same version everywhere.

**Comment by GarthDB** [comment]
Patches can be inconsistent numbers.

**Comment by GarthDB** [comment]
@timkim it looks good to me, do you want me to accept it?

**Comment by timkim** [comment]
Yes please.

**Pull request merged**